circle.yml

```diff
@@ -8,7 +8,8 @@ dependencies:
     - nvm install 4
     - nvm install 6
     - nvm install 8
-    - rm -rf 'node_modules coverage .nyc_output coverage.lcov'
+    - rm -rf node_modules ; rm -rf  coverage ; rm -rf .nyc_output ; rm -f coverage.lcov
```

Comment:
No need to separate these with semicolons. The single command will work fine.

Reply:
@stephenmathieson So removed those semicolons and surprisingly this time CircleCI accepted these traditional unix like cmd. Just bit clueless why it didn't get execute with same thing on my previous build before i added those semicolons and inverted commas .

As your suggestion reverted back the readme with it's original state removing all codecov related text etc.

regarding ava tap i followed the reference from here actually - https://github.com/istanbuljs/nyc.
In fact they referred to use it in same way like i did and i have been directed there by codecov nyc example . So as reference they say - 

```
update the scripts in your package.json to include these bins:
{
   "scripts": {
     "test": "nyc tap ./test/*.js",
     "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov"
   }
}
Many testing frameworks (Mocha, Tape, Tap, etc.) can produce TAP output. tap-nyc is a TAP formatter designed to look nice with nyc.
```
I ran without tap param too however it generates the same output but without a nice serial output formation. 

If you think we don't need it here certainly i can omit as it's not mandatory. Please suggest